core/controllers/reader.py

```diff
@@ -1283,16 +1282,72 @@ class RecommendationsHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'collection_id': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_regex_matched',
+                        'regex_pattern': constants.ENTITY_ID_REGEX
+                    }]
+                },
+                'default_value': None
+            },
+            'include_system_recommendations': {
+                'schema': {
+                    'type': 'bool'
+                },
+                'default_value': True
+            },
+            'stringified_author_recommended_ids': {
```

Comment:
Should the param be renamed, it is not really a stringified list anymore.

Reply:
Some params, `stringified_exp_ids` and `stringified_collection_ids`, have `stringified`. Others don't, like `selected_skill_ids`, `selected_subtopic_ids`, `mock_list`. Do you think it is good to delete all `stringified` in this PR?